packages/web-client/src/helpers/share/permission.ts

```diff
@@ -40,26 +40,30 @@ export abstract class SharePermissions {
     $gettext('Internal')
   )
 
-  static readonly read = new SharePermission('read', SharePermissionBit.Read, $gettext('Read'))
+  static readonly read = new SharePermission('read', SharePermissionBit.Read, $gettext('can read'))
```

Comment:
@kulmann could this be renamed to `can view` too? Or is there a real difference to `can view` ?

Reply:
Yes, good spot, please rename to `can view` :-)